shakenfist/db.py

```diff
@@ -50,6 +50,13 @@ def get_nodes():
     return etcd.get_all('node', None)
 
 
+def get_network_node():
+    see_this_node()
+    for n in get_nodes():
+        if n['ip'] == config.parsed.get('NETWORK_NODE_IP'):
+            return n
+
+
```

Comment:
This is fine for here, but it leaves me feeling like the current way of specifying if a node is a network node or not is clunky. I should think more about that.